doc/liblouis.texi

```diff
@@ -950,26 +962,23 @@ begemphword underline 456-36
 @end example
 
 @opcode{endemphword, <emphasis class>  <dot pattern>}
-Generally emphasis with word context ends when the word ends. However
-when an indication is required to close a word emphasis then this
-opcode defines the Braille dot pattern that indicates the end of a word
-emphasis.
+Word emphasis ends implicitly when the word ends. When an indication
+is required to close word emphasis, i.e.@: when emphasis ends in the
+middle of a word, then this opcode defines the braille dot pattern
+that is used.
 
 @example
 endemphword transnote 6-3
 @end example
 
-If emphasis ends in the middle of a word the Braille dot pattern
-defined in this opcode is also used.
-
 @opcode{emphmodechars, characters}
 
 Normally, only space characters will cancel the
 @opcoderef{begemphword} indicator. However, by using the
-@code{emphmodechars} opcode, you can specify the list of characters
-that are legal within a emphasized word. If @code{emphmodechars} is
-specified, any character that is not in this list and is not a
-@code{letter} will cancel the @opcoderef{begemphword} indicator.
+@code{emphmodechars}, you can specify the list of characters that are
```

Comment:
You removed the 'opcode' here. I think it should be here. Maybe you thought `@code{foo}` will generate 'foo opcode'? No it doesn't, only `@opcoderef{foo}^ will generate 'foo opcode'.

Reply:
Sorry, I indeed thought I saw a `@opcoderef`. I was too fast.